cdap-docs/reference-manual/source/licenses/cdap-level-1-dependencies.csv

```diff
@@ -1,6 +1,9 @@
 ch.qos.logback.logback-classic-1.0.9.jar,ch.qos.logback,logback-classic
 ch.qos.logback.logback-core-1.0.9.jar,ch.qos.logback,logback-core
+com.amazonaws.aws-java-sdk-bom-1.11.133.pom,com.amazonaws,aws-java-sdk-bom
```

Comment:
@sreevatsanraman This is the first `pom` dependency which we have added in license `com.amazonaws.aws-java-sdk-bom-1.11.133.pom`. I am not sure if whether we document this or not.
Talked to @anwar6953 about the need of this and according to him this dependency is not strictly need but removing it now might need some EMR testing. 

I have added for now since in the final rst and pdf the pom part gets removed anyways.